tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -145,7 +149,63 @@ public function doSomething($foo)
 }
 EOF;
 
-        $this->doTest($expected, $input);
+        yield [$expected, $input];
+
+        $expected = <<<'EOF'
+<?php namespace Space1 {
+    use Foo\Bar\Foo;
+    use Symfony\Annotation\Template;
+}
+
+namespace Space2 { use A,B; }
+
+namespace Space3 {
+    use Symfony\Annotation\Template;
+    use Symfony\Doctrine\Entities\Entity0, Zoo\Bar;
```

Comment:
Why like this?

```diff
-    use Zoo\Bar;
-    use Symfony\Annotation\Template, Symfony\Doctrine\Entities\Entity0;
+    use Symfony\Annotation\Template;
+    use Symfony\Doctrine\Entities\Entity0, Zoo\Bar;
```

IMHO it should be rather:

```diff
-    use Zoo\Bar;
-    use Symfony\Annotation\Template, Symfony\Doctrine\Entities\Entity0;
+    use Symfony\Annotation\Template, Symfony\Doctrine\Entities\Entity0;
+    use Zoo\Bar;
```

or split to 3-line import with proper order.

Reply:
This is current behavior of the fixer, to prevent making the PR even bigger I wouldn't change this